python_modules/libraries/dagster-dbt/dagster_dbt/asset_utils.py

```diff
@@ -367,16 +367,18 @@ def default_metadata_from_dbt_resource_props(
     metadata: Dict[str, Any] = {}
     columns = dbt_resource_props.get("columns", {})
     if len(columns) > 0:
-        metadata["columns"] = MetadataValue.table_schema(
-            TableSchema(
-                columns=[
-                    TableColumn(
-                        name=column_name,
-                        type=column_info.get("data_type") or "?",
-                        description=column_info.get("description"),
-                    )
-                    for column_name, column_info in columns.items()
-                ]
+        return dict(
+            TableMetadataEntries(
+                column_schema=TableSchema(
+                    columns=[
+                        TableColumn(
+                            name=column_name,
+                            type=column_info.get("data_type") or "?",
+                            description=column_info.get("description"),
+                        )
+                        for column_name, column_info in columns.items()
+                    ]
+                )
```

Comment:
Minor thing but we should have an opinion on it since this will be copy and pasted forever.

Should we standardize on


```python
        return dict(
            TableMetadataEntries(
                column_schema=TableSchema(
                    columns=[
                        TableColumn(
                            name=column_name,
                            type=column_info.get("data_type") or "?",
                            description=column_info.get("description"),
                        )
                        for column_name, column_info in columns.items()
                    ]
                )
             )
```
OR using Pydantic's `dict` method?

```python
        
            TableMetadataEntries(
                column_schema=TableSchema(
                    columns=[
                        TableColumn(
                            name=column_name,
                            type=column_info.get("data_type") or "?",
                            description=column_info.get("description"),
                        )
                        for column_name, column_info in columns.items()
                    ]
                ).dict()
```

Reply:
👁️👁️  - `.dict()` results in less indentation, so changed to that.